Wearable/src/main/java/com/vorsk/minimalin/model/ConfigData.java

```diff
@@ -148,6 +148,16 @@ public static Class getWatchFaceServiceClass() {
                         DEFAULT_SHOW_HANDS);
         settingsConfigData.add(settingShowHandsItem);
 
+        // Show bigger digits UX (toggle) in settings Activity.
+        ConfigItemType settingBiggerDigitsItem =
+                new SwitchConfigItem(
+                        context.getString(R.string.config_bigger_digits),
+                        R.drawable.clock,
```

Comment:
It would be better to get some different icons.
I like to use the [Material design icons](https://material.io/resources/icons/?style=baseline) and if I can't find anything good there I search [The Noun Project](https://thenounproject.com/).

Reply:
Thanks for the pointers. Used "zoom" icons for now.